scripts/gulpfiles/git_tasks.js

```diff
@@ -35,12 +35,10 @@ let upstream = null;
  */
 function getUpstream() {
   if (upstream) return upstream;
-  for (const line of String(execSync('git remote -v')).split('\n')) {
-    const [remote, url] = line.split('\t');
-    if (url.includes('github.com/google/blockly')) {
-      upstream = remote;
-      return upstream;
-    }
+  const remote = String(execSync('git remote -v | grep "google/blockly" | cut -f1 | head -1'));
+  if (remote) {
+    upstream = remote;
+    return upstream;
```

Comment:
This works, but it shouldn't be necessary to call a bunch of shell tools.  What about:

```suggestion
  for (const line of String(execSync('git remote -v')).split('\n')) {
    if (line.includes('google/blockly')) {
      return line.split('\t')[0];
    }
```

Otherwise: please fix line length.  (Maybe it's time we prettify the gulp scripts…)